rewrite-xml/src/main/java/org/openrewrite/xml/trait/SpringReference.java

```diff
@@ -26,13 +26,11 @@
 import org.openrewrite.xml.XPathMatcher;
 import org.openrewrite.xml.tree.Xml;
 
-import java.util.HashSet;
 import java.util.Optional;
-import java.util.Set;
 import java.util.regex.Pattern;
 
 @Value
-class SpringReference implements Reference {
+public class SpringReference implements Reference {
```

Comment:
As far as I'm aware only the Provider needs to be public

Reply:
I think you are right, but the inner `Provider` class will get a IntelliJ unused marker as soon as I remove the `public` keyword here. I think it is clearer to keep it public for that reason.